debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java

```diff
@@ -335,6 +335,14 @@ public void initializeStorage() {
         super.initializeStorage();
 
         try (AdminClient admin = AdminClient.create(this.producerConfig.asProperties())) {
+            boolean dbHistoryTopicExists = admin.listTopics().names()
```

Comment:
This looks good to me; my only question is whether there's a significant overhead in doing this, increasing connector start-up time. Do you have any feeling how long this call typically takes?

@jpechane, any objections to this chang from your POV?

Reply:
For typical setups where Kafka Connect and Kafka are deployed in the same datacenter and for small amounts of topics ~100 the call takes ~500ms to respond, 
but then when you start having ~1000-2000 topics, then the latency spikes to ~3000ms and sometimes will fail because the timeout is at 3s. Of course, as I increased in topics the latency increased too.

The same technique is also used in the `exists()` method of the KafkaDatabaseHistory class, where the get will block for `default.api.timeout.ms` which is 60s if not set. At least, in that case, the timeout is configurable.

So to answer the question, this adds significantly to the startup time.

I tried also `describeTopics` method and I'm getting much better performance even for many topics (< 200ms), so it's a better fit for this use case. I'll have to rework my PR.